tldr.py

```diff
@@ -41,14 +41,18 @@
     URLOPEN_CONTEXT.check_hostname = False
     URLOPEN_CONTEXT.verify_mode = ssl.CERT_NONE
 
-OS_DIRECTORIES = {
-    "linux": "linux",
+ OS_DIRECTORIES = {
```

Comment:
```suggestion
OS_DIRECTORIES = {
```
Is the space a typo?

Reply:
let's test it